site/content/docs/user/rootless.md

```diff
@@ -9,57 +9,214 @@ menu:
+
+Your host will also need to enable [cgroup delegation](https://systemd.io/CGROUP_DELEGATION/) of the `cpu` controller for
+user services. This is enabled by default for distributions running `systemd` version 252 and higher.
+
+To enable cgroup delegation for all the controllers, do the following:
+
+1. Check your version of `systemd` by running `systemctl --version`. If the output prints
+   `systemd 252` or higher, no further action is needed. Example output below from a Fedora host:
+
+   ```sh
+   $ systemctl --version
+   systemd 257 (257.9-2.fc42)
+   ```
+
+2. For systems with older versions of `systemd`, first create the directory
+   `/etc/systemd/system/user@.service.d/` if it is not present.
+
+   ```sh
+   sudo mdkir -p /etc/systemd/system/user@.service.d/
+   ```
+
+3. Next, create the file `/etc/systemd/system/user@.service.d/delegate.conf` with the following content:
+
+   ```ini
+   [Service]
+   Delegate=yes
+   ```
+
+4. Reload systemd for these changes to take effect:
+
+   ```sh
+   sudo systemctl daemon-reload
+   ```
+
+5. If using docker, reload the user docker daemon:
+
+   ```sh
+   systemctl --user restart docker
+   ```
+
+### Networking
+
+Containers running in rootless mode may not loaded with host-level iptable modules.
+This breaks the behavior of most networking components, such as Ingress and Gateway controllers.
+
+To load the iptable modules, do the following:
+
+1. First, use `lsmod` to check which kernel modules are loaded by default for user processes on
+   your system. Use `grep` to find which iptable modules are loaded:
+
+   ```sh
+   lsmod | grep "ip.*table"
+   ```
+
+2. Check the output for the following kernel modules:
+   - `ip6_tables`
+   - `ip6table_nat`
+   - `ip_tables`
+   - `iptable_nat`
+
+3. If one or more of the kernel modules above are not present, your system needs to load these at
+   startup for each process. First, run the following command to add these missing modules:
+   
+   ```sh
+   sudo tee /etc/modules-load.d/iptables.conf > /dev/null <<'EOF'
+   ip6_tables
+   ip6table_nat
+   ip_tables
+   iptable_nat
+   EOF
+   ```
 
-- Create `/etc/systemd/system/user@.service.d/delegate.conf` with the following content, and then run `sudo systemctl daemon-reload`:
+4. Check that the new module loading configuration is correct. You should see the following output:
 
-  ```ini
-  [Service]
-  Delegate=yes
-  ```
+   ```sh
+   $ cat /etc/modules-load.d/iptables.conf 
+   ip6_tables
+   ip6table_nat
+   ip_tables
+   iptable_nat
+   ```
 
-  (This is not enabled by default because ["the runtime impact of
-  [delegating the "cpu" controller] is still too
-  high"](https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/thread/ZMKLS7SHMRJLJ57NZCYPBAQ3UOYULV65/).
-  Beware that changing this configuration may affect system
-  performance.)
+4. Next, restart the `systemd-modules-load` service to make these changes effective immediately:
```

Comment:
`4` is used twice. Most markdown renderers handle that fine, but would be good to be consistent by either making them all `1` to be obvious that the numbering is left to the rendering step, or to have them be sequentially numbered.